.circleci/config.yml

```diff
@@ -22,6 +22,19 @@ windows_defaults: &windows_defaults
   executor:
     name: win/default
 
+test_matrix_unix: &test_matrix_unix
+  matrix:
+    parameters:
+      node_version: ['12', '16']
+      jdk_version: ['11.0.16-amzn']
```

Comment:
should we test jdk8?

Reply:
we have been testing it, here https://github.com/snyk/snyk-mvn-plugin/blob/master/.circleci/config.yml#L218 for example, probably worth retaining the current test versions if possible and then in a separate PR review whether those are appropriate, so this PR just does one thing, introduce matrix testing